kong/clustering/services/sync/rpc.lua

```diff
@@ -244,7 +244,8 @@ local function do_sync()
 
       local crud_event_type = "create"
 
-      if old_entity then
+      -- If we need to wipe lmdb, we dont need to delete old entity.
+      if old_entity and not ns_delta.wipe then
```

Comment:
Not correct, we should update `crud_event_type` in this if statement.

Reply:
fixed.